src/components/Buttons/TextButton.tsx

```diff
@@ -27,8 +27,8 @@ const StyledTextButton = styled.button`
   background: transparent;
   font-size: inherit;
   color: ${({ theme, disabled }) => {
-    if (disabled) return theme.textSecondary
-    else return theme.link
+    if (disabled) return theme.font.secondary
+    else return theme.global.accent
```

Comment:
nit: I know you just changed the values, but let's fix this to:

```ts
color: ${({ theme, disabled }) => disabled ? theme.font.secondary : theme.global.accent};
```

Reply:
I see a thumbs up but not a change, which is fine, it's a nit! Just curious why the thumbs up then 😅

Reply:
ah, probably because you changed it in the second place of this file where it was needed.